gridcapa-core-cc-post-processing-app/src/main/java/com/farao_community/farao/core_cc_post_processing/app/PostProcessingService.java

```diff
@@ -274,4 +276,34 @@ private void uploadF305ToMinio(String targetMinioFolder, ResponseMessageType res
             throw new CoreCCPostProcessingInternalException(String.format("Exception occurred while uploading F305 for business date %s", localDate), e);
         }
     }
+
+    private void zipRaoResultsAndSendToOutputs(String targetMinioFolder, Map<TaskDto, ProcessFileDto> raoResults, LocalDate localDate) {
+        String raoResultZipTmpDir = "/tmp/raoResults_out/" + localDate.toString() + "/";
+
+        // Add all raoResult json files from minio to tmp folder
+        raoResults.values().stream().filter(processFileDto -> processFileDto.getProcessFileStatus().equals(ProcessFileStatus.VALIDATED)).forEach(raoResult -> {
+            InputStream inputStream = minioAdapter.getFileFromFullPath(raoResult.getFilePath());
```

Comment:
We should use try-with-resources to ensure all closeable objects are properly closed (I see no `close()` in this method).